packages/fhir-converter/src/templates/cda/Resources/Observation.hbs

```diff
@@ -16,18 +16,22 @@
             {{/each}}
         ],
         
-        {{#if observationCategory}}
-             "category": [
-                  {
-                      "coding": [
-                          {
-                            "system": "http://terminology.hl7.org/CodeSystem/observation-category",
-                            "code": "{{observationCategory}}"
-                          }
-                    ]
-                  }
-            ],
-        {{/if}}
+            {{#if observationCategory}}
+                {{#with (evaluate 'Utils/ObservationCategoryDisplayFromCode.hbs' code=observationCategory) as |respObj|}}
+                    "category": [
+                        {
+                            "coding": [
+                                {
+                                    "system": "http://terminology.hl7.org/CodeSystem/observation-category",
+                                    "code": "{{../observationCategory}}",
+                                    "display": "{{respObj.display}}"
+                                }
+                            ],
+                            "text": "{{respObj.display}}"
```

Comment:
are we sure that wrapping `{{respObj.display}}` in quotes won't double quote it?

the output obj already returns a quoted string? or does it work because it's returning an object